recipes-tag/pysmbc/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set version = "1.0.14" %}
+package:
+  name: pysmbc
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hamano/pysmbc.git
+  git_rev: {{ version }}
+
+build:
+  number: 1
+  script: python setup.py install
```

Comment:
We might as well follow conda-forge's pattern and use `python setup.py install --single-version-externally-managed --record=record.txt`